### DISCUSSION TITLE
Gruppe kann nicht angelegt werden

### DISCUSSION THREAD

**Discussion by gost4711** [OpenCCU/OpenCCU#2125] [🇩🇪 Fragen&Antworten] [answered]
Hi,

ich habe Version 3.65.11.20221218 installiert und folgendes Problem.

Ich kann eine leere Gruppe mit "HmIP-Heizungssteuerung" anlegen. Sehe auch die IP-Geräte und kann diese auch der Gruppe hinzufügen. Sobald ich mit ok bestätige rödelt das System ohne Ende - zeigt "Bitte warten. Die direkten Verknüpfungen werden konfiguriert." - und es geschieht nix, egal wie lange ich warte! Das gleiche auch, wenn ich versuche die Gruppe mit einzelnen Geräte zu erstellen oder eine neue Gruppe erstelle und beim Erstellen die Geräte einfüge. Derzeit gehe ich den Weg die Geräte und dessen Kanäle händisch direkt verknüpfe - was gut funktioniert. Ich glaube aber das das nicht zielführend ist.

GOst4711

**Comment by jens-maus** [comment, marked as answer]
Problem bekannt und auch schon längst für die nächste Version repariert. Siehe:

https://github.com/jens-maus/RaspberryMatic/issues/2105

Einfach zur Vorversion zurückgehen oder auf die nächste Version warten.

**Reply by gost4711** [reply]
Vielen Dank, dann werde ich mal waren.

**Reply by sirvolcano** [reply]
@jens-maus: 
Sorry erstmal, dass ich gestern diesen Thread (https://github.com/jens-maus/RaspberryMatic/issues/945#issuecomment-1366092660) unwissentlich gehighjackt habe, war kein Absicht. Wie gesagt, Neuling und muss noch die Regeln lernen!

@gost4711:
Danke, dass Du eine neue Abfrage für das Problem geöffnet hast!

Dann werde ich mal schauen, wie ich das repariert bekomme. #2105 ist ja schon mal ein guter Hinweis! Danke!
